tests/common_tests/test_utils.py

```diff
@@ -55,15 +55,16 @@ class TestException(Exception):
 class TestUtils:
     """Test suite"""
 
-    def test_ensure_version_higher(self, caplog):
+    def test_ensure_version_higher(self, caplog, pg_db_conn):
         """Test version ensuring with bigger number"""
         utils.CFG.minimal_schema = 20
+        CursorMockVersion.version = 22
         utils.DatabasePoolConnection = DbPoolMock
         with caplog.at_level(logging.INFO):
             utils.ensure_minimal_schema_version()
-        assert f"Current schema version: {ACTUAL_SCHEMA_VERSION}, minimal required: {utils.CFG.minimal_schema}, OK" in caplog.messages[0]
+        assert f"Current schema version: {CursorMockVersion.version}, minimal required: {utils.CFG.minimal_schema}, OK" in caplog.messages[0]
```

Comment:
I'm not sure what this test does but why do you need to mock cursor and db pool? Does it even test the db `select` made by the `ensure_minimal_schema_version`? you have the 'real' initialized DB running in your tests (`pg_db_conn` should be it), just compare the current schema version to `sys.maxsize` and to `0` or `-sys.maxsize`. I'm missing some context here obviously, it's probably correct, it just caught my eye why it's so complicated.